test/performance/range/views/view_translate_1D_benchmark.cpp

```diff
@@ -0,0 +1,170 @@
+// -----------------------------------------------------------------------------------------------------
+// Copyright (c) 2006-2019, Knut Reinert & Freie Universität Berlin
+// Copyright (c) 2016-2019, Knut Reinert & MPI für molekulare Genetik
+// This file may be used, modified and/or redistributed under the terms of the 3-clause BSD-License
+// shipped with this file and also available at: https://github.com/seqan/seqan3/blob/master/LICENSE.md
+// -----------------------------------------------------------------------------------------------------
+
+#include <random>
+#include <vector>
+
+#include <benchmark/benchmark.h>
+
+#include <seqan3/range/views/translate.hpp>
+#include <seqan3/range/views/to.hpp>
+#include <seqan3/test/performance/sequence_generator.hpp>
+#include <seqan3/test/seqan2.hpp>
+#include <seqan3/alphabet/nucleotide/dna4.hpp>
+
+#ifdef SEQAN3_HAS_SEQAN2
+    #include <seqan/sequence.h>
+    #include <seqan/seq_io.h>
+    #include <seqan/translation.h>
+#endif
+
+using namespace seqan3;
+using namespace seqan3::test;
+
+struct baseline_tag{};
+struct translate_tag{};
+
+// ============================================================================
+//  sequential_read
+// ============================================================================
+
+template <typename rng_t>
+void sequential_read_impl(benchmark::State & state, rng_t && rng)
+{
+    for (auto _ : state)
+    {
+        for (auto && outer : rng)
```

Comment:
```suggestion
        for (aa27 c : rng)
```

If the type is simple, it is always better to write it. Also copy might be slightly better than && for alphabets.